src/main/java/com/rocketden/main/service/UserService.java

```diff
@@ -37,6 +38,18 @@ public UserDto createUser(CreateUserRequest request) {
         return UserMapper.toDto(user);
     }
 
+    public UserDto deleteUser(DeleteUserRequest request) {
+        User user = repository.findUserByNickname(request.getNickname());
+
+        if (user == null) {
```

Comment:
nit, personal preference: Add a comment above this `if` statement, similar to the `createUser(...)` method above. (No problem if you don't do this, just a personal preference.)